tests/ServerTest.php

```diff
@@ -51,7 +51,7 @@ public function getResultCases()
         $response = new Response();
 
         yield [$response, $response];
-        yield [function () use ($response) { return $response; }, $response];
+        yield [static function () use ($response) { return $response; }, $response];
```

Comment:
Please enable the `static_lambda` rule in `.php_cs.dist` as well.

Reply:
Done